insights_dart/lib/src/event.dart

```diff
@@ -1,115 +1,203 @@
-abstract class InsightsEvent {
-  String get eventName;
+enum EventType { click, conversion, view }
 
-  String get indexName;
+class Event {
```

Comment:
This class is part if the public API, it should be exported (and documented)

Reply:
Fixed